bin/pulsar

```diff
@@ -257,8 +257,8 @@ if [ -z "$PULSAR_LOG_CONF" ]; then
     PULSAR_LOG_CONF=$DEFAULT_LOG_CONF
 fi
 
-if [ -z "$PULSAR_PRESTO_CONF" ]; then
-    PULSAR_PRESTO_CONF=$DEFAULT_PULSAR_PRESTO_CONF
+if [ -z "$PULSAR_TRINO_CONF" ]; then
```

Comment:
for better handling retrocompatibility we could still check PULSAR_PRESTO_CONF as fallback. something like this:

```
if [ -z "$PULSAR_TRINO_CONF" ]; then
  if [ -z "$PULSAR_PRESTO_CONF" ]; then
     PULSAR_TRINO_CONF=$DEFAULT_PULSAR_TRINO_CONF
   else
     PULSAR_TRINO_CONF=$PULSAR_PRESTO_CONF
   fi
fi
```

Reply:
Implemented in 43c8c7da41154289baf4fd26e568ce2ace4ae451.

Reply:
Oops. I think I misunderstand your suggestion, followup as 362207d3e54369175bca2553609f10ba0855ca7d.

Then a version upgrade should go well, while a downgrade needs a caveat. And anyway, PIP-200 publicly drop the previous configure so that we can drop it in the following versions. This is for a smoothly deprecating period.